code/datums/emotes.dm

```diff
@@ -111,14 +113,14 @@
 		. = message_larva
 	else if(isAI(user) && message_AI)
 		. = message_AI
+	else if(isMoMMI(user) && message_mommi)
+		. = message_mommi
```

Comment:
moved because MoMMIs pass issilicon() checks so it'd always use message_silicon if both message_silicon and message_mommi are defined